nerfstudio/data/dataparsers/wod_utils.py

```diff
@@ -0,0 +1,697 @@
+from __future__ import annotations
+
+import warnings
+from copy import deepcopy
+from typing import Optional
+
+# Disable annoying warnings from PyArrow using under the hood.
+warnings.simplefilter(action="ignore", category=FutureWarning)
+import glob
+import os
+from dataclasses import asdict, dataclass
+from typing import Dict, List, Literal, Tuple, Type, TypedDict
+
+import dask.dataframe as dd
+import numpy as np
+import numpy.typing as npt
+import tensorflow as tf
+tf.config.set_visible_devices([], 'GPU') # Not useful for parsing data. 
+import transforms3d
+from tqdm import tqdm
+from typing_extensions import NotRequired
+from waymo_open_dataset import v2
+from waymo_open_dataset.utils import box_utils, transform_utils
+from waymo_open_dataset.v2.perception import box as _v2_box
+from waymo_open_dataset.v2.perception import camera_image as _v2_camera_image
+from waymo_open_dataset.v2.perception import context as _v2_context
+from waymo_open_dataset.v2.perception import lidar as _v2_lidar
+from waymo_open_dataset.v2.perception import pose as _v2_pose
+from waymo_open_dataset.v2.perception.utils.lidar_utils import convert_range_image_to_cartesian
+from waymo_open_dataset.wdl_limited.camera.ops import py_camera_model_ops
+
+WOD64_ELEVATION_MAPPING = {
```

Comment:
could you put this in `lidar_elevation_mapping.py` instead?

Reply:
Yep this could be changed. Will do that.